src/sky.h

```diff
@@ -79,6 +83,131 @@ class Sky : public scene::ISceneNode
 		m_skycolor = skycolor;
 	}
 	void setBodiesVisible(bool visible) { m_bodies_visible = visible; }
+	void setSunVisible(bool sun_visible) { m_sun_visible = sun_visible; }
+	void setSunGlowVisible(bool sun_glow) { m_sun_glow = sun_glow; }
+	void setMoonVisible(bool moon_visible) { m_moon_visible = moon_visible; }
+	void setStarsVisible(bool stars_visible) { m_stars_visible = stars_visible; }
+	void setMeshVisible(bool dynamic_visible) { m_dynamic_visible = dynamic_visible; }
+	void setStarCount(u16 star_count)
+	{ 	
+		if (star_count != m_star_count) {
```

Comment:
`if (star_count == m_star_count) return;` (with newline in between)  reduces the amount of indents.